ckanext/dia/harvester/dcat.py

```diff
@@ -36,6 +39,11 @@ class DIADCATJSONHarvester(DCATJSONHarvester):
         }
     ''')
 
+    # IDCATRDFHarvester
+    def update_session(self, session):
+        session.headers.update({'X-Harvest': 'data.govt.nz/dcat-json'})
+        return session
```

Comment:
See https://github.com/ckan/ckanext-dcat/blob/568952af6f7f1f4801449b2d1c6f0b0df15ba5f8/ckanext/dcat/harvesters/base.py#L63 for how this extension hook is managed. Note that the DIADCATJSONHarvester extends DCATJSONHarvester which extends DCATHarvester (which defines the extension hook so that we can access/modify the request headers).